src/Illuminate/Database/Schema/Blueprint.php

```diff
@@ -631,9 +631,9 @@ public function nullableTimestamps()
 	 */
 	public function timestamps()
 	{
-		$this->timestamp('created_at');
+		$this->timestamp('created_at')->default(\DB::raw('CURRENT_TIMESTAMP'));
 
-		$this->timestamp('updated_at');
+		$this->timestamp('updated_at')->default(\DB::raw('CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP'));
```

Comment:
@clabinger 

> I believe the normal Laravel behavior is to automatically update the created_at and updated_at fields when we update a record with Eloquent. But if we manually update a database row without going through Eloquent, the timestamp fields will not be updated.

Hmm, interesting... I've had a deeper dig into the code in the newer versions of the framework. It seems like actually they are simply setting the column to be nullable(). So the default would be null.

This might be a better option here as I assume it would still leave Laravel in charge of setting or not setting them.

But in addition, I know that there is now an option within eloquent to ignore the timestamps when creating or updating. Think it's just `withoutTimestamps()`.

Reply:
Having the default be null sounds good to me since that is in line with current Laravel functionality.

`withoutTimestamps()` was added in 9.31 ([see here](https://github.com/laravel/framework/compare/v9.30.0...v9.31.0)) and I confirmed that it was already using `null` as the default value as of 5.3.

Reply:
Thanks @clabinger - I've pushed an update to reflect this discussion.
Please check!